order-summary/run.js

```diff
@@ -79,8 +80,21 @@ fs.readdir("./samples", function (err, files) {
 
 /** Helper to pull the url from the api data based on variant id */
 function getURLforID(id, apiData) {
-  let url = apiData.find(o => o.number == id)
-  return (url.images[0].url)
+  const currentItem = apiData.find(o => o.number == id)
+  let url
+  if (currentItem){
+    const firstImage = currentItem.images[0]
```

Comment:
note this can be null - so the call below to `firstImage.url` may fail - also should not define this until it is needed (if imageWithCheckoutTag comes back, we never use this)

Reply:
this is fixed in last commit